app/public/css/styles.css

```diff
@@ -38,7 +54,7 @@ html {
 body {
   margin: 0;
   padding: 0;
-  font-family: sans-serif;
+  font-family: 'JTMarnie', sans-serif, arial;
```

Comment:
The font family `sans-serif` must be the last. This way having `arial` at the end will not help.

Reply:
```suggestion
  font-family: JTMarnie, Arial, Helvetica, sans-serif;
```

Reply:
You don't need to use quotes around name in this case.

https://developer.mozilla.org/en-US/docs/Web/CSS/font-family#Valid_family_names